nilearn/input_data/nifti_masker.py

```diff
@@ -186,7 +186,7 @@ def fit(self, imgs=None, y=None):
             self.affine_ = self.mask_img_.get_affine()
         # Load data in memory
         self.mask_img_.get_data()
-        if self.verbose > 10:
+        if self.verbose > 0:
```

Comment:
I am not too sure whether > 10 was a typo or meant to indicate a super high verbosity was needed. At least the required verbosity is the same as the other print statements in the same class so that's probably fine.